src/sentry/api/endpoints/organization_slugs.py

```diff
@@ -1,12 +1,21 @@
 from django.core.exceptions import ValidationError
-from django.core.validators import validate_slug
-from django.db import router, transaction
+from django.core.validators import RegexValidator
+from django.db import IntegrityError, router, transaction
+from django.utils.regex_helper import _lazy_re_compile
 from rest_framework.request import Request
 from rest_framework.response import Response
 
-from sentry.api.base import region_silo_endpoint
+from sentry.api.base import DEFAULT_SLUG_ERROR_MESSAGE, DEFAULT_SLUG_PATTERN, region_silo_endpoint
 from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.api.exceptions import ConflictError
 from sentry.models import Project
+from sentry.utils.snowflake import MaxSnowflakeRetryError
+
+_validate_sentry_slug = RegexValidator(
+    _lazy_re_compile(DEFAULT_SLUG_PATTERN),
+    DEFAULT_SLUG_ERROR_MESSAGE,
+    "invalid",
+)
```

Comment:
Copies format of original `django.core.validators.validate_slug` except the regex pattern and error message is different